src/ra.erl

```diff
@@ -796,7 +797,8 @@ local_query(ServerId, QueryFun, Timeout) ->
 %% @end
 -spec leader_query(ServerId :: ra_server_id(),
                    QueryFun :: query_fun()) ->
-    {ok, {ra_idxterm(), term()}, ra_server_id() | not_known}.
+    ra_server_proc:ra_leader_call_ret(term()) |
```

Comment:
Same as below, `{ok, Result, Leader::ra_server_id()}` is covered by `ra_server_proc:ra_leader_call_ret/1`.